Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.messages.in

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(GPU_PROCESS) && HAVE(AVASSETREADER)
 
+[EnabledBy=UseGPUProcessForMediaEnabled]
```

Comment:
Thinking more about it, it should probably be:
```suggestion
[EnabledBy=UseGPUProcessForMediaEnabled && MediaPlaybackEnabled]
```

Reply:
👍  Done